### DISCUSSION TITLE
Feature Request: Enable running specs with Firefox as the webdriver

### DISCUSSION THREAD

**Discussion by baarkerlounger** [activeadmin/activeadmin#7880] [Ideas]
Currently running the full test suite requires having Chrome installed to be used as the web driver. As this isn't the default browser installed on many linux distributions, being able to have the test suite run using Chrome or Firefox as the web driver would lower the barrier to getting this up and running so you can contribute.

It might also mean browser engine specific bugs are more likely to be caught.

**Comment by deivid-rodriguez** [comment]
Sure, please open a PR.

**Comment by javierjulio** [comment]
I would welcome a PR for this. Curious what would be available for this in Ruby. When I’ve dealt with JS I’ve used Playwright which is great.
